src/pages/docs/deprecations/index.md

```diff
@@ -23,6 +23,16 @@ Deprecations are subject to change in detail or time frame. If you need help ass
 
 ## Planned Deprecations
 
+## Deprecations for 2025.3
+
+### Removing support on Linux self-hosted and Cloud for SHA-1 certificates in Octopus Tentacles
+
+Starting in Octopus 2025.3, Tentacle instances using SHA‑1 certificates will no longer be supported. SHA‑1 is an outdated hashing algorithm with known security weaknesses, and modern security standards recommend stronger alternatives such as SHA‑256.
+
+If any of your deployment targets are running Tentacles with SHA‑1 certificates, they will be unable to connect to your Octopus Server after upgrading to 2025.3. We recommend updating your Tentacle certificates to SHA‑256 before upgrading.
```

Comment:
Simon's blog post said this

> We plan to release an in-product configuration option to turn on/off the SHA-1 Tentacle support in Octopus Server 2025.2 and remove support for SHA-1 Tentacles in the subsequent Octopus Server 2025.4 release. These changes will impact Octopus Server Linux container customers only.

which doesn't align with the 2025.3 messaging here. Which one is correct?

Reply:
The issue closing in 2025.3 is correct. It's unfortunate the blog post mixes the migration guide with some of these predicted plans and release numbers, but they are listed purely as plans, with blog posts being point-in-time. Whereas this deprecation guide provides the correct release numbers based of the closed issue, which is the source of truth in this case.